_posts/2024-02-17-social-integration.md

```diff
@@ -0,0 +1,170 @@
+This is a good license for an Open Source game, as it ensures that everyone who makes a modification to the game also have to release the source for that modification.
+And, if we would like to, we can integrate that change back into the vanilla game again, allowing everyone to enjoy that modification.
+
+There is one downside of the GPLv2 license: it is not really compatible with many other licenses, as it is kinda strict in the implications.
+I am not going to delve too much into the details in this post, but just know once something is GPLv2, everything it touches has to be GPLv2 or compatible with it.
+
+To integrate with social platforms like Steam, Discord, etc, you have to make use of an SDK they supply.
+As you might have guessed by now, they are not licensed under a so-called "free" license, which conflicts with our GPLv2 license.
+
+Besides the legal part of the license, we also have the social part of "how people feel about it", if we would integrate such non-free SDKs directly into the game.
+And this has always been a debate.
+
+As it goes in the world of Open Source Software, people tend to have a very strong opinion about the "free" part of the license.
+And so integrating with something non-free is against the principles, and as such, by their argument, bad.
+We can't simply dismiss such arguments because we want to integrate with social platforms.
+
+Which means we needed to find a balance: make sure OpenTTD as a game remains Open Source, free, and GPLv2 licensed.
+But also allow integration with social platforms.
+
+The solution, as it turns out?
+Make it a choice.
+
+## Plugin system
+
+To ensure the game itself remains licensed under GPLv2, we created a simple plugin system which allows us to extend the game with other binaries which might be licensed differently.
+We have been doing this for years and years now, via our [BaNaNaS system](https://bananas.openttd.org).
+This delivers in-game content to the user, while the author of such content can license it how ever they like.
+The only "demand" we make, is that the author gives us the right to distribute their content, and that it is available free of charge.
+
+In result, we see a wide variety of content on BaNaNaS, under all kinds of licenses.
+Some authors embrace the ideology of OpenTTD, and use open or free licenses.
+Others are more protective of their work, and do not allow anyone to make derivatives of their work.
+From our point of view, we don't actually mind either way: as long as our players can download it for free and enjoy the content, we are more than happy and grateful for your efforts.
+
+The plugin system that is added to 14.0 uses the same logic and mechanism.
+Although technically it is very different, more on that later, but from your point of view it is the same: download a plugin, put it in the right folder, and enjoy the added functionality.
+
+Currently we release three plugins: Steam, Discord, and GOG Galaxy.
```

Comment:
Should this sentence be rephrased to say, "Currently we are releasing three plugins: Steam, Discord, and GOG Galaxy."?